.github/workflows/examples-openiotsdk.yaml

```diff
@@ -154,6 +154,14 @@ jobs:
               run: |
                   scripts/examples/gn_build_example.sh examples/ota-provider-app/linux/ out/ota-provider chip_config_network_layer_ble=false
 
+            - name: Archive Map files
+              uses: actions/upload-artifact@v3
```

Comment:
I guess it's only for validation purposes.

Reply:
It make sense to keep these map files if we keep the crash handler enabled (I'd prefer that).

Reply:
Ok, I think it'll be hard to upstreaming this but we can keep it in our fork (internal changes). I think we can also add `if: ${{ failure() }}` condition to reduce the number of stored artifacts and put this at the end of the workflow.

Reply:
I'm changing this

Reply:
done